decorator.rb

```diff
@@ -0,0 +1,11 @@
+require '.\nameable'
```

Comment:
- [ ] Please kindly check my detailed comments on the `trimmer_decorator` file to effect changes suggested by the last reviewer here which I explained very well. 👍